midend/include/Dialect/Gemmini/Transform.h

```diff
@@ -38,6 +46,7 @@ typedef float acc_scale_t;
 typedef uint32_t scale_t_bits;
 typedef float scale_t;
 typedef int32_t scale_acc_t;
+typedef int32_t acc_t;
```

Comment:
It should not be added here `acc_t`, `DIM`,`ADDR_LEN`.

Reply:
"acc_t" has no usage. But ADDR_LEN and DIM are useful variables in LegalizeForLLVMExport.cpp. Should I transfer the "define" statement to LegalizeForLLVMExport.cpp?